src/Kmd.Logic.DocumentGeneration.Client/regenerate.ps1

```diff
@@ -1,4 +1,4 @@
-autorest --input-file=documentgeneration.swagger.json --output-folder=. --namespace=Kmd.Logic.DocumentGeneration.Client --csharp --override-client-name=InternalClient --add-credentials
+autorest --input-file=documentgeneration.swagger.json --output-folder=. --namespace=Kmd.Logic.DocumentGeneration.Client --csharp --legacy --override-client-name=InternalClient --add-credentials
```

Comment:
Can we add a comment here that why `--legacy` is used?

Reply:
Resolved